.gitmodules

```diff
@@ -9,6 +9,7 @@
 [submodule "third_party/double-conversion"]
 	path = third_party/double-conversion
 	url = https://github.com/google/double-conversion
+	branch = master
```

Comment:
Does this mean each `git submodule` will fetch the latest commit from `master` branch?
I think it is preferable to pin the commit to the current latest in this case.

Reply:
According to my understanding, it should not be pulled from the latest every time, but when adding submodules, a branch is specified.  it should be from the commit 256ac80956 of this branch at that time. Manually update this submodule. Maybe I don't know much about github.

Reply:
I think this should be `main` branch

Reply:
@nmanhong I think if you want to update the submodule, you can go into the directory and run `git fetch` and `git merge` the upstream branch to update the local code. In this case the upstream branch would be the `master` branch. AFAIK this doesn't need to be specified in the `.gitmodules` file.

Reply:
@nmanhong I'm happy to merge this once we've removed the following line from this PR.